$NetBSD$

* Changes from NetBSD base.
* This is a pre-generated file (from ntpd/ntpdbase-opts.def via AutoGen);
  patch it directly rather than the .def source, which would make the
  .def newer than the shipped generated files and trigger AutoGen
  regeneration at build time -- AutoGen is not a build dependency of
  this package.  Update the compiled-in default from /etc/ntp.conf to
  match this build's CONFIG_FILE (PR pkg/47149); driftfile and keyfile
  defaults are untouched since this patch does not change them.

--- ntpd/ntpd.1ntpdman.orig	2020-06-23 16:03:47.000000000 +0000
+++ ntpd/ntpd.1ntpdman
@@ -153,7 +153,7 @@ The name and path of the configuration file,
 configuration file name.
 .sp
 The name and path of the configuration file,
-\fI/etc/ntp.conf\fP
+\fI/usr/pkg/etc/ntp.conf\fP
 by default.
 .TP
 .NOP \f\*[B-Font]\-d\f[], \f\*[B-Font]\-\-debug\-level\f[]
@@ -176,7 +176,7 @@ configuration specification in the
 This is the same operation as the
 \fBdriftfile\fP \fIdriftfile\fP
 configuration specification in the
-\fI/etc/ntp.conf\fP
+\fI/usr/pkg/etc/ntp.conf\fP
 file.
 .TP
 .NOP \f\*[B-Font]\-g\f[], \f\*[B-Font]\-\-panicgate\f[]
@@ -900,7 +900,7 @@ See \fBOPTION PRESETS\fP for configuration environment
 See \fBOPTION PRESETS\fP for configuration environment variables.
 .SH FILES
 .TP 15
-.NOP \fI/etc/ntp.conf\f[]
+.NOP \fI/usr/pkg/etc/ntp.conf\f[]
 the default name of the configuration file
 .br
 .ns
